netlify.toml

```diff
@@ -0,0 +1,8 @@
+[build]
+    command = "npm run build"
+    publish = "/build"
+
+[[redirects]]
+    from = "/*"
+    to = "/index.html"
+    status = 200
```

Comment:
Good job deploying it even though it wasn't required! Was nice to see a live site 👍